code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -28,7 +28,7 @@
 	payday_modifier = 0.75
 	family_heirlooms = list(/obj/item/flashlight/lantern/heirloom_moth)
 
-/datum/species/moth/regenerate_organs(mob/living/carbon/C,datum/species/old_species,replace_current=TRUE,list/excluded_zones)
+/datum/species/moth/regenerate_organs(mob/living/carbon/C,datum/species/old_species,replace_current=TRUE,list/excluded_zones,visual_only)
```

Comment:
```suggestion
/datum/species/moth/regenerate_organs(mob/living/carbon/C, datum/species/old_species, replace_current= TRUE, list/excluded_zones, visual_only)
```
i know you only added the last part but wouldn't it be quirky if you updated the whoel thing

Reply:
It's a bit long, but sure I'm good with that, not long enough to be a problem